HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.at(0) == $F) { class.name.postln }
+}; "";
```

Comment:
I use ""; a lot to suppress default result posting when I want to print a list of stuff in the post window.
Why not leave it in and explain it? Documentation is for learning useful tricks too ;-)
```
// post a list of all classes which have a method called 'play'
Class.allClasses.select {|class| class.methods.detect { |meth| meth.name == \play }.notNil }.printAll;
""; // post blank string instead of resulting list to reduce post window clutter
```